src/index.html

```diff
@@ -248,7 +250,7 @@ <h1 class="c-slide__title col-md-6">Scale up your team with experienced develope
                 </section>
                 
                 <div class="l-bg">
-                    <img src="./img/c-loop-bg.jpg" alt="background-loop" class="c-loop-bg l-loop-bg">
+                    <img alt="background-loop" class="c-loop-bg l-loop-bg js-lazy-img" data-src="./img/c-loop-bg.jpg" data-loaded="false">
```

Comment:
Do we need to add `data-loaded=false`. IMHO we could add `data-loaded=true` from js